src/modules/commander/Commander.hpp

```diff
@@ -118,7 +121,14 @@ class Commander : public ModuleBase<Commander>, public ModuleParams
 		(ParamFloat<px4::params::COM_DISARM_LAND>) _param_com_disarm_land,
 
 		(ParamInt<px4::params::COM_OBS_AVOID>) _param_com_obs_avoid,
-		(ParamInt<px4::params::COM_OA_BOOT_T>) _param_com_oa_boot_t
+		(ParamInt<px4::params::COM_OA_BOOT_T>) _param_com_oa_boot_t,
+
+		(ParamFloat<px4::params::COM_TAS_FS_INNOV>) _tas_innov_threshold,
+		(ParamInt<px4::params::COM_TAS_FS_T1>) _tas_use_stop_delay,
```

Comment:
Can we change `COM_TAS_FS_T1`  and `COM_TAS_FS_T2` to something more descriptive?

Reply:
Here are some ideas, although I don't actually like any of these better than T1 and T2:

T1:
COM_TAS_FS_TIME
COM_TAS_FS_START
COM_TAS_FS_TOUT
COM_TAS_FS_TRIG

T2:
COM_TAS_FS_END
COM_TAS_FS_STOP

e.g. the confusion comes from is it the TAS that is starting/stopping for the FS that is starting/stopping?

Reply:
> I don't actually like any of these better than T1 and T2:

I (stubbornly) came to the same conclusion.